requirements/ci32_requirements.txt

```diff
@@ -1,3 +1,3 @@
 spin==0.15
 # Keep this in sync with ci_requirements.txt
-scipy-openblas32==0.3.30.0.1
+scipy-openblas32==0.3.30.0.6
```

Comment:
Note this does not change the base OpenBLAS commit used, only patches out the not-quite-ready code. OpenBLAS HEAD has also reverted that change, so the patch will not be needed for OpenBLAS 0.3.31. That is why the version here only changes the build from 1 to 6.